src/fu-engine.c

```diff
@@ -4380,7 +4380,7 @@ fu_engine_load_plugins (FuEngine *self, GError **error)
 		g_autoptr(GError) error_local = NULL;
 
 		/* ignore non-plugins */
-		if (!g_str_has_suffix (fn, ".so"))
+		if (!g_str_has_suffix (fn, G_MODULE_SUFFIX))
```

Comment:
Actually, thinking about this -- I think we need to add back the "." -- e.g. `"." G_MODULE_SUFFIX" otherwise we might match directories with the name `gutso` for example.

Reply:
Good point